src/iOS/toga_iOS/libs/uikit.py

```diff
@@ -289,10 +293,28 @@ def NSTextAlignment(alignment):
 # UITextField.h
 UITextField = ObjCClass('UITextField')
 
-UITextBorderStyleNone = 0
-UITextBorderStyleLine = 1
-UITextBorderStyleBezel = 2
-UITextBorderStyleRoundedRect = 3
+class UITextBorderStyle(Enum):
+    NoBorder = 0
+    Line = 1
+    Bezel = 2
+    RoundedRect = 3
+
+######################################################################
+# UITextInputTraits.h
+
+class UIKeyboardType(Enum):
```

Comment:
It would be so rad if we could generate this from calling Objc code. Again, not a blocker, more me thinking out loud.

Reply:
... and a pony :-)

Unfortunately, it's a C-level compiler constant, so there's absolutely nothing exposed about it's existence, range, scope, or anything. All you have to go on is documentation that says it exists. There's no runtime visibility.